src/safeds/data/tabular/transformation/_discretizer.py

```diff
@@ -0,0 +1,201 @@
+        self._column_names: list[str] | None = None
+        self._wrapped_transformer: sk_KBinsDiscretizer | None = None
+
+        if number_of_bins < 2:
+            raise ValueError("Parameter 'number_of_bins' must be >= 2.")
+        self.number_of_bins = number_of_bins
+
+    def fit(self, table: Table, column_names: list[str] | None) -> Discretizer:
+        """
+        Learn a transformation for a set of columns in a table.
+
+        This transformer is not modified.
+
+        Parameters
+        ----------
+        table : Table
+            The table used to fit the transformer.
+        column_names : Optional[list[str]]
+            The list of columns from the table used to fit the transformer. If `None`, all columns are used.
+
+        Returns
+        -------
+        fitted_transformer : TableTransformer
+            The fitted transformer.
+        """
+        if column_names is None:
+            column_names = table.column_names
+        else:
+            missing_columns = set(column_names) - set(table.column_names)
+            if len(missing_columns) > 0:
+                raise UnknownColumnNameError(list(missing_columns))
+
+        wrapped_transformer = sk_KBinsDiscretizer(n_bins=self.number_of_bins)
+        wrapped_transformer.fit(table._data[column_names])
+
+        result = Discretizer()
+        result._wrapped_transformer = wrapped_transformer
+        result._column_names = column_names
+
+        return result
+
+    def transform(self, table: Table) -> Table:
+        """
+        Apply the learned transformation to a table.
+
+        The table is not modified.
+
+        Parameters
+        ----------
+        table : Table
+            The table to which the learned transformation is applied.
+
+        Returns
+        -------
+        transformed_table : Table
+            The transformed table.
+
+        Raises
+        ------
+        TransformerNotFittedError
+            If the transformer has not been fitted yet.
+        """
+        # Transformer has not been fitted yet
+        if self._wrapped_transformer is None or self._column_names is None:
+            raise TransformerNotFittedError
+
+        # Input table does not contain all columns used to fit the transformer
+        missing_columns = set(self._column_names) - set(table.column_names)
+        if len(missing_columns) > 0:
+            raise UnknownColumnNameError(list(missing_columns))
+
+        data = table._data.copy()
+        data.columns = table.column_names
+        data[self._column_names] = self._wrapped_transformer.transform(data[self._column_names])
+        return Table._from_pandas_dataframe(data)
+
+    def inverse_transform(self, transformed_table: Table) -> Table:
+        """
+        Transform discretized data back to original feature space.
+
+        Note that this function does not regenerate the original data due to discretization rounding.
+        The table is not modified.
```

Comment:
```suggestion
        Undo the learned transformation.

        The table is not modified.
```
Keep these Docstrings as they should be the same for every transformer